Documentation/compatibility/Change-SignedXML-and-SignedCMS-default-algorithms-to-SHA256.md

```diff
@@ -0,0 +1,32 @@
+## Change SignedXML and SignedCMS default algorithms to SHA256
```

Comment:
Ah gotcha, I messed up the tense then. I should have made it past tense i.e.

"Chang**ed** the SignedXml and SignedCMS default algorithm to SHA256" where the subject is an implied "I" i.e. "I changed signedxml and signedcms..."

Yours is better because it avoids the implied subject issue. I'll change it. Thanks for the explanation.